rust/src/services/tests/fetchers.rs

```diff
@@ -110,13 +108,12 @@ async fn test_scalar_svc() {
     let resp = task.call(ScalarRequest).await;
     assert_eq!(resp, Ok(None));
 
-    let round_id = subscriber.params_listener().get_latest().round_id;
-    publisher.broadcast_scalar(round_id.clone(), ScalarUpdate::New(42.42));
+    publisher.broadcast_scalar(ScalarUpdate::New(42.42));
```

Comment:
this is violating the masking configuration, the scalar is truncated by the masking method to fit the config, but the coordinator shouldn't broadcast invalid values.
```suggestion
    publisher.broadcast_scalar(ScalarUpdate::New(0.42));
```